bld/namelist_files/namelist_defaults_ctsm.xml

```diff
@@ -1166,6 +1172,9 @@ lnd/clm2/surfdata_map/release-clm5.0.18/surfdata_1x1_numaIA_hist_78pfts_CMIP6_si
 <fsurdat hgrid="1x1_brazil"    sim_year="1850" use_crop=".true." >
 lnd/clm2/surfdata_map/release-clm5.0.18/surfdata_1x1_brazil_hist_78pfts_CMIP6_simyr1850_c190214.nc</fsurdat>
 
+<fsurdat hgrid="NEON">
+/glade/work/jedwards/sandboxes/NEON/inputdata/single_point/datmdata_NEON/${NEONSITE}/surfdata_0.9x1.25_hist_16pfts_Irrig_CMIP6_simyr2000_${NEONSITE}.nc</fsurdat>
```

Comment:
In order to use filenames with time-stamps in the names, this should be moved to the user_nl_clm file in the user-mod directory for each NEON site.

Reply:
Ahhh, and now I see this is superseded by the setting of fsurdat in the shared default user_nl_clm file below. So there really is no reason for this and it should be removed.